core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -153,6 +154,12 @@ private static <T> Supplier<T> memoizeWithExpiration(Supplier<T> s) {
     return () -> Suppliers.memoizeWithExpiration(s::get, 100, MILLISECONDS).get();
   }
 
+  private static <T> Supplier<T> memoizeWithExpiration(Supplier<T> s, long duration,
+      TimeUnit units) {
+    // This insanity exists to make modernizer plugin happy. We are living in the future now.
+    return () -> Suppliers.memoizeWithExpiration(s::get, duration, units).get();
+  }
+
```

Comment:
I am not sure this is actually needed. Also, the comment isn't very helpful. I was able to inline this easily and use a static import, and completely remove this method, as well as the method it was copied from. Modernizer is perfectly fine with it.

Reply:
This was removed in f8583e0 and I just used a call  `Suppliers.memoize`